### PR TITLE
Update version 3.8

### PR DESCRIPTION
**Pull request by acordonez**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by acordonez** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,8 +35,9 @@ requirements:
     - cdutil
     - cdp
     - matplotlib-base
-    - numpy =1.23.5
+    - numpy
```
Comment (lee1043) [inline recipe/meta.yaml]: Can this be like this: `numpy >=2.0.0,<3.0.0`

**Review by lee1043 — APPROVED** [review]

**Comment by acordonez** [comment]
I'm not sure why the environment isn't getting solved because I was able to create one from the dev.yml file included in the PMP code with no problem. ([link to error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1099995&view=logs&j=a60d4a33-7c5f-58bf-6ede-6cf61ef1d926&t=516b3d9b-14d8-5583-9c60-738ad10c6805&l=2064))

**Comment by lee1043** [comment]
```
   └─ xcdat [0.7.1|0.7.2] would require
      └─ numpy >=1.23.0,<2.0  with the potential options
```
This is the part of the error message, which I think it is coming from conflict between older xcdat and numpy > 2.0. Could you add xcdat >= 0.7.3 in the meta.yml file?

**Comment by lee1043** [comment]
@acordonez together with `xcdat >= 0.7.3`, could you please also add `eofs >= 2.0.0` which both are the versions comply with `numpy >= 2.0`?

**Comment by acordonez** [comment]
@lee1043 I think this might be the problem: ([link](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1100636&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736&l=567))
Apparently the package needs to get built with the runtime version of numpy and it is not defaulting to the correct value for the build ([link](https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html)). Doing some reading about how to specify the build version.

**Comment by acordonez** [comment]
Closing this to try again on a clean branch

**Pull request closed without merging**